controller/ca/controller.go

```diff
@@ -111,7 +111,7 @@ func (c *CertificateController) syncNamespace(ns string) error {
 	configMap := &v1.ConfigMap{
 		ObjectMeta: metav1.ObjectMeta{Name: pkgK8s.TLSTrustAnchorConfigMapName},
 		Data: map[string]string{
-			pkgK8s.TLSTrustAnchorFileName: c.ca.TrustAnchorPEM(),
+			pkgK8s.TLSTrustAnchorFileName: c.ca.Cred.Crt.EncodeCertificatePEM(),
```

Comment:
TIOLI since you're using embedded structs, you could shorten this to:

```suggestion
			pkgK8s.TLSTrustAnchorFileName: c.ca.Cred.EncodeCertificatePEM(),
```

Reply:
omg this is so much better